doc-content/jbpm-docs/src/main/asciidoc/Kafka/message-send-event-proc.adoc

```diff
@@ -19,13 +24,13 @@ endif::JBPM,DROOLS,OP[]
 . Select the business process and open the business process designer.
 . Add an intermediate throw event or an end event of the type `message`.
 . Open the properties of the event.
-. In the *Message* field, select *New* and then enter the name of the message. This name must be the same as the name of the topic to which the event is to send Kafka messages, or else must be defined in an `org.kie.server.jbpm-kafka.ext.topics._broker-topic-name_` {KIE_SERVER} system property.
+. In the *Message* field, select *New* and then enter the name of the message. This name must be the same as the name of the topic to which the event is to send Kafka messages, or else must be defined as `_topic-name_` in an `org.kie.server.jbpm-kafka.ext.topics._topic-name_` {KIE_SERVER} system property.
 +
 For instructions about using `org.kie.server.jbpm-kafka.ext.topics.*` system properties to define topic names, see xref:kieserver-kafka-proc_{context}[].
 +
-. Add an input data item. Select the data object that you created as its type.
+. Add an input data item and set the type of this data item to the data object that you created.
```

Comment:
[Style suggestion] - Rephrase to remove "of this" and "that you created":

s/Add an input data item and set the type of this data item to the data object that you created./Add an input data item and set the data type to match the data object.

Reply:
Same comment as before.